android/src/app/organicmaps/util/Utils.java

```diff
@@ -492,6 +493,18 @@ public static String getLanguageCode()
     return Locale.getDefault().getLanguage();
   }
 
+  @NonNull
+  public static String getDecimalSeparator()
+  {
+    return String.valueOf(DecimalFormatSymbols.getInstance().getDecimalSeparator());
```

Comment:
I converted both decimal and grouping separators to string so the jni code was the same as for the country and language code. I've already modified the code so instead of string, we use chars.